### PR TITLE
Fix to valve state (device _TZE204_mpbki2zm)

### PR DESCRIPTION
**Pull request by prsveloso**

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -6354,7 +6354,7 @@ const definitions: Definition[] = [
                 [27, 'local_temperature_calibration', tuya.valueConverter.localTemperatureCalibration],
                 [28, 'fan_mode', tuya.valueConverterBasic.lookup(
                     {'low': tuya.enum(0), 'medium': tuya.enum(1), 'high': tuya.enum(2), 'auto': tuya.enum(3)})],
-                [36, 'valve', tuya.valueConverterBasic.lookup({'OPEN': 0, 'CLOSE': 1})],
+                [36, 'valve', tuya.valueConverterBasic.lookup({'ON': 0, 'OFF': 1})],
```
Comment (Koenkk) [inline src/devices/tuya.ts:6357]: `OPEN`/`CLOSE` makes more sense to me, why did you change this?

Comment (prsveloso) [reply src/devices/tuya.ts:6357]: For me too, but it doesn't work in Home-Assistant. Valve state entity wasn't changing. I see it changing correctly in Zigbee2MQTT Add-on but not in HA.
The change came from this (see megabitus98 post at near end): [https://github.com/Koenkk/zigbee-herdsman-converters/pull/6174](url)

Comment (Koenkk) [reply src/devices/tuya.ts:6357]: - Does it work after this change?
- Can you provide the debug log when changing from HA and it doesn't work?
See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable debug logging.

Comment (prsveloso) [reply src/devices/tuya.ts:6357]: Hi, @Koenkk 

Yes, it works after this change.
At the moment I can't provide the log, but I have this mqtt log:
```typescript
2023-11-09 15:24:49.215 DEBUG (MainThread) [homeassistant.components.mqtt.models] Rendering incoming payload '{"child_lock":"UNLOCK","current_heating_setpoint":28,"deadzone_temperature":1,"eco_mode":"OFF","fan_mode":"auto","linkquality":63,"local_temperature":21.5,"local_temperature_calibration":-3,"manual_mode":"ON","max_temperature":35,"max_temperature_limit":20,"min_temperature":10,"min_temperature_limit":26,"schedule_friday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_monday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_saturday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_sunday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_thursday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_tuesday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_wednesday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","state":"ON","system_mode":"heat","valve":"CLOSE"}' with variables {'entity_id': 'binary_sensor.0xa4c138d2d0efcf1c_valve', 'name': 'Valve', 'this': <template TemplateStateFromEntityId(binary_sensor.0xa4c138d2d0efcf1c_valve)>} and Template<template=({{ value_json.valve }}) renders=151>
2023-11-09 15:24:49.215 INFO (MainThread) [homeassistant.components.mqtt.binary_sensor] No matching payload found for entity: binary_sensor.0xa4c138d2d0efcf1c_valve with state topic: zigbee2mqtt/0xa4c138d2d0efcf1c. Payload: '{"child_lock":"UNLOCK","current_heating_setpoint":28,"deadzone_temperature":1,"eco_mode":"OFF","fan_mode":"auto","linkquality":63,"local_temperature":21.5,"local_temperature_calibration":-3,"manual_mode":"ON","max_temperature":35,"max_temperature_limit":20,"min_temperature":10,"min_temperature_limit":26,"schedule_friday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_monday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_saturday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_sunday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_thursday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_tuesday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_wednesday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","state":"ON","system_mode":"heat","valve":"CLOSE"}', template output: 'CLOSE', with value template 'Template<template=({{ value_json.valve }}) renders=152>'
2023-11-09 15:24:49.216 DEBUG (MainThread) [homeassistant.components.mqtt.models] Rendering incoming payload '{"child_lock":"UNLOCK","current_heating_setpoint":28,"deadzone_temperature":1,"eco_mode":"OFF","fan_mode":"auto","linkquality":63,"local_temperature":21.5,"local_temperature_calibration":-3,"manual_mode":"ON","max_temperature":35,"max_temperature_limit":20,"min_temperature":10,"min_temperature_limit":26,"schedule_friday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_monday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_saturday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_sunday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_thursday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_tuesday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","schedule_wednesday":"06:00/20.0 11:30/20.0 13:30/20.0 17:30/20.0","state":"ON","system_mode":"heat","valve":"CLOSE"}' with variables {'entity_id': 'text.0xa4c138d2d0efcf1c_schedule_monday', 'name': 'Schedule monday', 'this': <template TemplateStateFromEntityId(text.0xa4c138d2d0efcf1c_schedule_monday)>} and Template<template=({{ value_json.schedule_monday }}) renders=151>
```

Comment (megabitus98) [reply src/devices/tuya.ts:6357]: Hi @prsveloso,
I've tried now to change the word CLOSED to CLOSE, and this also seems to work in HA.
`e.binary('valve', ea.STATE, 'OPEN', 'CLOSE').withDescription('3-Way Valve status'),`
`[36, 'valve', tuya.valueConverterBasic.lookup({'OPEN': 0, 'CLOSE': 1})],`

Comment (prsveloso) [reply src/devices/tuya.ts:6357]: I confirm that it works too @megabitus98. Awesome!
@Koenkk can you please dismiss this PR and change only this line? 
```
e.binary('valve', ea.STATE, 'OPEN', 'CLOSE').withDescription('3-Way Valve status'),
``` 
Thank you both!

Comment (Koenkk) [reply src/devices/tuya.ts:6357]: @prsveloso please update your pr by clicking [here](https://github.com/prsveloso/zigbee-herdsman-converters/edit/master/src/devices/tuya.ts?pr=%2FKoenkk%2Fzigbee-herdsman-converters%2Fpull%2F6440)

Comment (prsveloso) [reply src/devices/tuya.ts]: Done!
Thanks.

**Comment by Koenkk** [comment]
Thanks!
